File: benchmark/tests/threads-single/threads_001.php
<?php

define('L', 10000000);

$tini = microtime(true);
function proc($ini, $fim)
{
	$acc = 0;
	$i = $ini;
	while ($i <= $fim) {
		$acc += $i++;
	}
	return $acc;
}

$r1 = proc(0, 2 * L);

$tfim = microtime(true);

$t1 = ($tfim - $tini);

print "PHP\n";
print $r1;

File: benchmark/tests/threads-single/threads_001.c
#include <stdlib.h>
#include <time.h>
#include <stdio.h>

int main()
{
	
	time_t t0 = clock();

	int L = 10000000;
	long long acc = 0;
	int ini = 0, fim = 2 * L;
	while (ini <= fim) acc += ini++;
	
	time_t t1 = clock();

	printf("C\n");
	printf("%lld\n", acc);
	return 0;
}

